awot/io/read_t28.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import absolute_import, print_function
+import numpy as np
+from netCDF4 import num2date
+from .flight import _winduv
+
+
+def read_t28_netcdf(ncFile):
+    """
+    T-28 has enough differences from other aircraft netCDF files to
+    warrant its own read script.
+
```

Comment:
Yeah, I agree there is code duplication. At the same time, I have effectively cordoned off the T-28 routines to their own code chunk. There is merit in just having customized read scripts since aircraft data are kind of all over the place in terms of formats.

Up to you what you want to do. All I care is the T-28 is read into flattened arrays. This io script is inelegant in some ways (not all), but it works.